config/database.yml

```diff
@@ -21,11 +21,13 @@ default: &default
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   username: postgres
+  password: postgres
```

Comment:
revert that changes from `database.yml` as these are your local :)

Reply:
Reverted, my fault :disappointed: